main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -2410,6 +2411,40 @@ object Parser2 {
       close(mark, TreeKind.Expr.UnsafeOld)
     }
 
+    /**
+      * `unsafe_remove TTYPE {EXPRESSION}` or `unsafe_remove {TTYPE} EXPRESSION`
+      *
+      * produces
+      *
+      *   - TreeKind.Expr.Unsafe
+      *     - TreeKind.Type.Type
+      *     - TreeKind.Expr.Expr
```

Comment:
In the general case, it would also say here what nodes can be errors or missing, but in this case they are not. Doing this in general would make weeder/parser more readable since you just have to be faithful to the comment on both sides